sqlitecluster/SQLiteNode.cpp

```diff
@@ -862,6 +864,16 @@ bool SQLiteNode::update() {
             }
         }
 
+        // If the majority of logged in full peers responds with abstain, then re-search.
+        bool majorityAbstained = abstainCount * 2 >= numLoggedInFullPeers;
```

Comment:
It shouldn't be logged in full peers, but full peers regardless of whether they're logged in.

I.e., if there are 5 full peers, you should only count this as abstained when three nodes abstained, even if you're only currently logged into one other node.

Reply:
Ah, I think I misunderstood what "logged in" means. What does it mean exactly?

Reply:
"logged in" means "currently connected and communicating". The first thing nodes do when they come online is attempt to log in to the other nodes.